docs/source/upgrade_to_newest_version.md

```diff
@@ -40,6 +43,19 @@ For v1.4 Nodejs chaincode libraries, the supported node runtime is v8. Though no
 Also please note that the v1.4 libraries themselves are not supported, and will no longer get updates. Please migrate to a v2 Node chaincode library.
 
 For detailed information please refer to the [compatibility](https://github.com/hyperledger/fabric-chaincode-node/blob/main/COMPATIBILITY.md) document in the `fabric-chaincode-node` repository.
+
+#### Chaincode initialization
+
+The new chaincode lifecycle does not have an option to automatically call a chaincode `Init` function upon chaincode deployment.
+Instead, the new chaincode lifecycle has a mechanism to flag that chaincode initialization is required,
+by using the `--init-required` flag, and providing a `--isInit` flag on the `peer chaincode invoke` command.
+However, is most scenarios it is recommended to embed initialization logic into chaincode rather than use the chaincode lifecycle mechanism.
```

Comment:
```suggestion
However, in most scenarios it is recommended to embed initialization logic into chaincode rather than use the chaincode lifecycle mechanism.
```